omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -913,581 +1133,1838 @@
+                                                        </div>
+
+                                                        <div class="col-3 duration">
+                                                            <label>Number of Days</label>
+                                                            <obs conceptId="159368"/>
+                                                        </div>
+                                                    </div>
+                                                </obsgroup>
                                             </div>
                                         </div>
-                                    </div>
-                                    <div class="card">
-                                        <div class="card-header">Diagnosis</div>
-                                        <div class="card-body">
-                                            <div class ="row">
-                                                <div class ="col-md-12">
-                                                    <encounterDiagnoses required="false" selectedDiagnosesTarget="" preferredCodingSource="ICD-10-WHO"/>
-                                                </div>
+
+                                        <div class="card" style="margin-top:10px">
+                                            <div class="card-header">
+                                                Fluconazole
+                                            </div>
+                                            <div class="card-body">
+                                                <obsgroup groupingConceptId="165306">
+                                                    <div class="row">
+                                                        <div class="col-3">
+                                                            <br/>
+                                                            <obs conceptId="1282" answerConceptSetIds="165308"
+                                                                 answerConceptIds="747" style="radio"
+                                                                 class="horizontal"/>
+                                                        </div>
+                                                        <div class="col-2">
+                                                            <obs labelText="No. of Pills" conceptId="160856"/>
+                                                        </div>
+                                                        <div class="col-2 duration">
+                                                            <obs conceptId="159368" labelText="No. of Days"/>
+                                                        </div>
+
+                                                        <div class="col-2">
+                                                            <obs id="br" conceptId="1190" labelText="Start Date"/>
```

Comment:
@slubwama Remove all references of id="br" from the old AIJAR forms as this was cleaned up

Reply:
done